dom.bs

```diff
@@ -6484,8 +6514,19 @@ string <var>namespace</var> (default null):</p>
 
  <li><p>If <var>oldAttr</var> is <var>attr</var>, return <var>attr</var>.
 
- <li><p>If <var>oldAttr</var> is non-null, then <a lt="replace an attribute">replace</a>
- <var>oldAttr</var> with <var>attr</var>.
+ <li>
+  <p>If <var>oldAttr</var> is non-null, then:</p>
```

Comment:
Would this condition result in the TT-check being skipped if the old attribute didn't exist?

Reply:
In that case append an attribute is called which runs validate and set as it's first step, so TT check still happens

Reply:
I need to check if replace is used elsewhere, if it's not I might move the validate and set call into it, to be consistent with other places.

Reply:
I don't think replace is used anywhere else so will move the validation step a level down.